webui/react/src/e2e/models/common/base/BasePage.ts

```diff
@@ -1,70 +1,10 @@
-import { Locator, type Page } from '@playwright/test';
+import { BasePage as BP } from 'playwright-page-model-base/BasePage';
 
 import { Navigation } from 'e2e/models/components/Navigation';
 
-export interface ModelBasics {
-  get pwLocator(): Locator;
-}
-
 /**
  * Base model for any Page in src/pages/
  */
-export abstract class BasePage implements ModelBasics {
-  readonly _page: Page;
+export abstract class BasePage extends BP {
```

Comment:
Having BasePage extend BasePage is confusing to me. I wouldn't know which to pull in if I were new to this. I think anything called `BaseThing` reads as shared. Can this be DeterminedPage` or `DeterminedBasePage` or something along those lines?  It would help make it clear that this is determined specific.